applications/GeoMechanicsApplication/tests/cpp_tests/test_geomechanics_time_integration_scheme.cpp

```diff
@@ -38,10 +38,7 @@ class ConcreteGeoMechanicsTimeIntegrationScheme
     }
 
 protected:
-    void UpdateVariablesDerivatives(ModelPart& rModelPart) override
-    {
-        // Intentionally left empty
-    }
+    MOCK_METHOD(void, UpdateVariablesDerivatives, (ModelPart & rModelPart), (override));
```

Comment:
It seems like you can discard the function parameter name here (which helps to get a nicer formatting):
```suggestion
    MOCK_METHOD(void, UpdateVariablesDerivatives, (ModelPart &), (override));
```